custom_components/circadian_lighting/__init__.py

```diff
@@ -164,6 +164,10 @@ def get_sunrise_sunset(self, date = None):
         else:
             import astral
             location = astral.Location()
```

Comment:
You may want to drop this old line, since your try-except block replaces this. Gave me this otherwise:

```
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/setup.py", line 248, in _async_setup_component
    result = await task
  File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/circadian_lighting/__init__.py", line 109, in setup
    cl = CircadianLighting(hass, min_colortemp, max_colortemp,
  File "/config/custom_components/circadian_lighting/__init__.py", line 139, in __init__
    self.data['percent'] = self.calc_percent()
  File "/config/custom_components/circadian_lighting/__init__.py", line 220, in calc_percent
    today_sun_times = self.get_sunrise_sunset(now)
  File "/config/custom_components/circadian_lighting/__init__.py", line 174, in get_sunrise_sunset
    location = astral.Location()
AttributeError: module 'astral' has no attribute 'Location'
```